### PR TITLE
Add better support for negative numbers

### PR DESCRIPTION
**Pull request by codymoorhouse**

### CODE REVIEW

File: components/Moola.vue [hunk 1]
```diff
@@ -43,11 +43,6 @@
             value: {
                 type: [ Number, String ],
                 default: 0
-            },
-
-            delimiter: {
```
Comment (kotarCreative) [inline components/Moola.vue]: Why are you removing support for custom delimiters?

Comment (codymoorhouse) [reply components/Moola.vue]: Copy and paste error i believe

File: components/Moola.vue [hunk 2]
```diff
@@ -103,13 +103,18 @@
                 return split.join('.');
             },
 
-            checkDecimals(event) {
+            blur(event) {
                 var val = this.removeCommas(event.target.value);
```
Comment (kotarCreative) [inline components/Moola.vue]: This should probably be renamed to `removeDelimiters`

File: components/Moola.vue [hunk 3]
```diff
@@ -103,13 +103,18 @@
...
                 var el = this.$refs['moola'];
 
-                el.value = this.addCommas(Number(val).toFixed(this.precision));
-                this.$emit('input', Number(val).toFixed(this.precision));
+                var emitVal = Number(val).toFixed(this.precision);
+
+                if (isNaN(emitVal) || !isFinite(emitVal)) {
+                    emitVal = "0.00";
+                }
+                el.value = this.addCommas(emitVal);
```
Comment (kotarCreative) [inline components/Moola.vue]: This should probably be renamed to `addDelimiters`

Comment (codymoorhouse) [reply components/Moola.vue]: Changed!

File: components/Moola.vue (hunk 2 above)
Comment (codymoorhouse) [reply components/Moola.vue]: Changed!

**Pull request merged**